src/main/java/io/confluent/idesidecar/restapi/messageviewer/strategy/ConfluentCloudConsumeStrategy.java

```diff
@@ -222,6 +225,37 @@ private PartitionConsumeData processPartition(
     );
   }
 
+  private List<SimpleConsumeMultiPartitionResponse.PartitionConsumeRecordHeader>
+  decodeHeaders(PartitionConsumeRecord record) {
+    return record
+        .headers()
+        .stream()
+        .map(header -> {
+          String decodedValue;
+          try {
+            decodedValue = new String(
+                BASE64_DECODER.decode(header.value()), StandardCharsets.UTF_8
+            );
+          } catch (IllegalArgumentException e) {
+            // For whatever reason, we couldn't decode the Base64 encoded header value.
+            // Perhaps the API changed or the data is corrupted. Either way,
+            // we log a warning and return the raw value.
+            Log.debugf(e, "Failed to base64 decode header value '%s' from Confluent Cloud" +
+                    "(partition: %d, offset: %d)",
+                header.value(), record.partitionId(), record.offset()
+            );
```

Comment:
uber-nit:

```suggestion
            // we log a debug message and return the raw value.
            Log.debugf(
                e,
                "Failed to base64 decode header value '%s' from Confluent Cloud "
                    + "(partition: %d, offset: %d)",
                header.value(),
                record.partitionId(),
                record.offset()
            );
```

Reply:
Ha, all good. I've addressed it. Thanks!